lifecycle-livedata-ktx/src/main/kotlin/Transformations.kt

```diff
@@ -5,6 +5,6 @@ import androidx.lifecycle.distinctUntilChanged
 import androidx.lifecycle.map
 
 /** Sequentially calls [map] and [distinctUntilChanged]. */
-public inline fun <X, Y> LiveData<X>.mapDistinct(crossinline transform: (X) -> Y): LiveData<Y> {
+public fun <X, Y> LiveData<X>.mapDistinct(transform: (X) -> Y): LiveData<Y> {
```

Comment:
Will it be possible to keep this inline if we make lambda signature the same as in lambda passed to `map`?

Reply:
nope

![image](https://github.com/RedMadRobot/redmadrobot-android-ktx/assets/18189767/5644b822-9bf9-4226-9ed8-ae71cac52266)